### PR TITLE
Add Transparency Report H2 2022 (Fixes #12748)

### LINKED ISSUE #12748: Transparency Report H2 2022
## Description

Needs to be published by Friday 17th Feb

---

#### Success Criteria
- [ ] TP is published by Friday 17th Feb

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds Transparency Report page for H2 2022. The EU MAU numbers have a hard legal deadline, so this needs to go live tomorrow (Friday 17th Feb).

## Issue / Bugzilla link

#12748

## Testing

http://localhost:8000/en-US/about/policy/transparency/jul-dec-2022/

### CODE REVIEW

**Comment by alexgibson** [comment]
This is ready for review, but Do not Merge until Friday 17th Feb

**Review by reemhamz — APPROVED** [review]
Super clean work!
r+ 🍭

I'll leave it for you to merge in case you want to wait till tomorrow :)

**Comment by reemhamz** [comment]
Oop, I forgot to check for mobile in my review.
I noticed that in some tables, there is overflow on the right for smaller screens (starting at `385px`).

I think the horizontal scrolling here may be a necessity because of how the table is structured. Looks like the previous years have also had the same "feature" 😉

But yeah, just wanted to flag it down!

Screen Shot 2023-02-16 at 9 26 01 PM

**Comment by alexgibson** [comment]
Got the go-ahead to merge / push this today

**Pull request merged**
